x/ccv/provider/keeper/relay.go

```diff
@@ -103,8 +103,18 @@ func (k Keeper) completeMaturedUnbondingOps(ctx sdk.Context) {
 		// Attempt to complete unbonding in staking module
 		err := k.stakingKeeper.UnbondingCanComplete(ctx, id)
 		if err != nil {
-			// UnbondingCanComplete fails if the unbonding operation was not found,
-			// which means that the state of the x/staking module of cosmos-sdk is invalid.
+			if stakingtypes.ErrUnbondingNotFound.Is(err) {
+				// The unbonding was not found.
```

Comment:
It's a bit unclear to me why `k.stakingKeeper.UnbondingCanComplete` would return `stakingtypes.ErrUnbondingNotFound` while `k.stakingKeeper.GetUnbondingType(ctx, id)` would return found as true?

Reply:
Have a look at https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics/x/staking/keeper/unbonding.go#L218